pod/annotations.go

```diff
@@ -234,6 +234,11 @@ const (
 	AnnotationKeyPodParameterMockPodPrepareTime = "mockPod.netflix.com/prepareTime"
 	AnnotationKeyPodParameterMockPodRunTime     = "mockPod.netflix.com/runTime"
 	AnnotationKeyPodParameterMockPodKillTime    = "mockPod.netflix.com/killTime"
+
+	// version recording
+	AnnotationKeyRuntimeVersion                    = "runtime.version.titus.netflix.com"
+	AnnotationKeySuffixRuntimeVersionTitusExecutor = "titus-executor"
```

Comment:
If we are doing just 2 here, I would prefer that the annotation be fully written out so it can be greppable.

Heck, I think all annotation constants should be fully written out! DRY be damned!